Makefile

```diff
@@ -173,7 +173,6 @@ analyze: $(JS)
 ifeq ($(DEV), ashell)
 	@cat fragments/prj.mdef.dev >> prj.mdef
```

Comment:
yes, we don't need to .mdef file at all. The heap pool can be defined in .conf file as well (CONFIG_HEAP_MEM_POOL_SIZE)